semgrep_output_v1.atd

```diff
@@ -145,6 +145,15 @@ type match_severity
   | Info <json name="INFO">
   | Experiment <json name="EXPERIMENT">
   | Inventory <json name="INVENTORY">
+  (* since 1.71.0, meant to replace the cases above where
+   * Error -> High, Warning -> Medium, and Info -> Low,
+   * and Experiment and Inventory should be removed.
```

Comment:
Is there more information on the removal of Experiment and Inventory? I mapped these to "low" on the app side (marked findings that had severity "experiment" as "low" severity.

Reply:
I think Inventory was used for CAI (Code Asset Inventory). I don't think we used it anymore. Maybe @p4p3r knows.
Experiment, I'm not sure what it is. Again maybe @p4p3r or @minusworld remember.

Reply:
Code/SR doesn't use these values at the moment but it would be useful to keep one of them (experiment?) because we'd like to revive the experimentation framework at some point.

Reply:
I think that SSC maybe still uses experiment based on looking at app code. But if not, this would be a great opportunity IMO to get rid of both of these fields and if we want to spin up an experimentation framework down the line do it without overloading "severity". cc @misha-semgrep for insight maybe on the SSC Experiment severity